lib/testing/CMakeLists.txt

```diff
@@ -14,7 +14,9 @@ add_library(
     gpuSimulateCudaBasebandBeamformer.cpp
     gpuSimulateCudaFRBBeamformer.cpp
     gpuSimulateCudaUpchannelize.cpp
-    gpuSimulateN2k.cpp
+    gpuSimulateN2kCorr.cpp
```

Comment:
So the file is called `gpuSimulate`, and what it does is to perform the actual (correct, valid) calculation on a CPU? Would it make sense to rename `gpuSimulate` to `cpu`?